utils/file.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+import json
+
+
+def create_json(data, json_file):
+    path = Path(f"data/{json_file}")
+    if path.is_file():
+        with open(path, "a") as file:
+            json.dump(data, file, indent=2)
+            file.write('\n')
+            return f'The data is added to the <{json_file}> file'
+    else:
+        with open(path, "w") as file:
+            json.dump(data, file, indent=2)
+            file.write('\n')
+            return f'created <{json_file}> file and data written to it'
+        
```

Comment:
truksta print result eilutes

Reply:
tai isveda sita f'created <{json_file}> file and data written to it' ar pakeist i print vietoj return?

Reply:
turi printint pranesima ivykus sekmingam scrapinimui,realei uzduotis nieko saugot nepraso nebent cia del saves tik

Reply:
del saves pabandymai